src/util.rs

```diff
@@ -0,0 +1,69 @@
+use crate::state::ValidatorMetrics;
+use cosmwasm_bignumber::Decimal256;
+use cosmwasm_std::{Decimal, Uint128};
+
+pub fn decimal_summation_in_256(a: Decimal, b: Decimal) -> Decimal {
+    let a_u256: Decimal256 = a.into();
+    let b_u256: Decimal256 = b.into();
+    let c_u256: Decimal = (b_u256 + a_u256).into();
+    c_u256
+}
+
+pub fn clamp(min: u32, value: u32, max: u32) -> u32 {
+    if value < min {
+        min
+    } else if value > max {
+        max
+    } else {
+        value
+    }
+}
+
+pub fn decimal_subtraction_in_256(a: Decimal, b: Decimal) -> Decimal {
+    let a_u256: Decimal256 = a.into();
+    let b_u256: Decimal256 = b.into();
+    let c_u256: Decimal = (a_u256 - b_u256).into();
+    c_u256
+}
+
+pub fn decimal_multiplication_in_256(a: Decimal, b: Decimal) -> Decimal {
+    let a_u256: Decimal256 = a.into();
+    let b_u256: Decimal256 = b.into();
+    let c_u256: Decimal = (b_u256 * a_u256).into();
+    c_u256
+}
+
+pub fn decimal_division_in_256(a: Decimal, b: Decimal) -> Decimal {
```

Comment:
Where are we checking for b!=0?

Reply:
It's a runtime error anyway right?

Reply:
We still want some meaningful error there. Not a unexpected failure or something.